packages/api/src/error-handler.js

```diff
@@ -0,0 +1,56 @@
+import { ErrorCode as MagicErrors } from '@magic-sdk/admin'
+import { JSONResponse } from './utils/json-response.js'
+import { HTTPError, UserNotFoundError, TokenNotFoundError } from './errors.js'
+
+/**
+ * @param {Error & {status?: number;code?: string;}} err
+ */
+export function errorHandler (err) {
+  let error = {
+    code: err.code,
+    message: err.message
+  }
+  let status = err.status || 500
+
+  if (err instanceof HTTPError || err instanceof UserNotFoundError || err instanceof TokenNotFoundError) {
```

Comment:
`err instanceof HTTPError` should be true when `err` is a `UserNotFoundError` or `TokenNotFoundError`.